build/salkafka/Dockerfile

```diff
@@ -16,11 +16,15 @@ RUN chown saluser:saluser /home/saluser/.startup.sh && \
 
 USER saluser
 
+RUN source /home/saluser/.setup_sal_env.sh && \
```

Comment:
What was the reason this call was split out from where it was previously?

Reply:
Just to make the layers leaner and reduce rebuilding steps. The way it was before, every time I changed salkafka or idl version it would reinstall kafkit and aiokafka.